packages/core-kit/src/language/name.ts

```diff
@@ -0,0 +1,832 @@
+/* eslint-disable max-lines */
+import { LanguageKey } from './type';
+
+// TODO: @Guamian @Betty correct and provide fully-translated names later
+export const languageNames: Readonly<Record<LanguageKey, string>> = Object.freeze({
```

Comment:
Please explain why we should commit these generated content to the repo. Even if you have a valid reason it should not commit to this package since the size is large.